strings/base_includes.h

```diff
@@ -10,6 +10,7 @@
 #include <map>
 #include <memory>
 #include <optional>
+#include <ostream>
```

Comment:
Is there no way to use a feature macro rather than including this in every single project? That seems problematic.

Reply:
Maybe this could be included only if WINRT_LEAN_AND_MEAN isn't defined. Or should I define a separate macro?

Reply:
Using `WINRT_LEAN_AND_MEAN` seems like a good idea. 👍

Reply:
Done.